src/app/pages/admin/user-details/user-details.component.ts

```
@@ -73,23 +74,23 @@ export class UserDetailsComponent implements OnInit {
     const userId = this.user()?.user_id;
     if (!userId) {
       console.error('No user ID available');
-      this.errorNotification.set('No user ID available');
-      this.hideNotificationsAfterDelay();
+      this.errorAlert.set('No user ID available');
+      this.hideAlertsAfterDelay();
```

I'm not sure if we should be hiding error notifications after a delay? Keeping around so the user can see them helps them understand what's happening (and maybe report issues).

---

That’s a good point, I think the “common standard” duration for alerts is usually around 5-10 seconds, but keeping error notifications visible to help users makes sense as well. I’ll go ahead and remove the auto-hide.